docs/application/dotnet/guides/location-sensors/device-sensors.md

```diff
@@ -185,6 +186,22 @@ The following table provides information about the accelerometer output for a de
 | Y-polarity     | +                                        | 0                                        | -                                        | 0                                        | 0                                        | 0                                        |
 | Z-polarity     | 0                                        | 0                                        | 0                                        | 0                                        | +                                        | -                                        |
 
+
+ <a name="auto_rotation"></a>
+## AutoRotation Sensor
+
+The autorotation sensor is the software sensor which uses the accelerometer to compute the orientation of the device. This sensor helps to determine if the device is placed in a landscape or portrait orientation.
+
+**Table: Measurement data detected by the AutoRotation sensor**
+
+| Measurement | Type                     | Range                         | Unit         |
+|-----------|------------------------|-----------------------------|------------|
+| TimeSpan   | `TimeSpan`     | -                             | Microseconds |
+| Accuracy    | `SensorDataAccuracy` | -                             | int          |
+| Rotation   | `AutoRotationState` | -     | -            |
+
+The `AutoRotationState` property is one of the values of the [Tizen.Sensor.AutoRotationState](https://samsung.github.io/TizenFX/latest/api/Tizen.Sensor.AutoRotationState.html) enumeration: `Degree_0`, `Degree_90`, `Degree_180`, or `Degree_270`.
```

Comment:
Just for check.
For now the link is broken. Is it because of the unpublished api version?

Reply:
@mijin85cho To be honest, I don't know what should i do at this stage. Should I create a document for that link? `AutoRotationState` merged with the master branch quite a long time ago.

Reply:
With your comment, I checked the test link, https://samsung.github.io/TizenFX/API7/api/Tizen.Sensor.AutoRotationState.html.

The api link with the API level 7 is valid, so I think when Tizen 6.0 is published, the URL with "latest" will get valid also. 
For now the link is broken, but as it will work properly when Tizen 6.0 is published, you don't need to do anything further for the link.

Reply:
Thanks for the suggestion @Reni-90 
However, this format seems to be used in other parts also.
So how about keeping the colon in this format for this page only for the consistency with other parts of this page?

![image](https://user-images.githubusercontent.com/9958374/82870549-191efe00-9f6b-11ea-9ab2-7fe7e878acbd.png)

![image](https://user-images.githubusercontent.com/9958374/82870567-2340fc80-9f6b-11ea-9890-fd985a616b2e.png)

![image](https://user-images.githubusercontent.com/9958374/82870586-2c31ce00-9f6b-11ea-9488-386e492940a3.png)

Reply:
@mijin85cho 
Yes, if we change it in one place, we need to change all the other parts as well.
So only for the consistency for this page, we can keep it as it is.

Reply:
Thanks for understanding @Reni-90